MaterialDesignThemes.Wpf/Themes/MaterialDesign3.ToggleButton.xaml

```diff
@@ -633,34 +628,34 @@
     <Setter Property="Validation.ErrorTemplate" Value="{StaticResource MaterialDesignValidationErrorTemplate}" />
     <Setter Property="Width" Value="52" />
     <Setter Property="wpf:ToggleButtonAssist.OnContent" Value="{StaticResource SwitchCheckMarkIcon}" />
-    <Setter Property="wpf:ToggleButtonAssist.SwitchTrackOffBackground" Value="{DynamicResource MaterialDesignCardBackground}" />
-    <Setter Property="wpf:ToggleButtonAssist.SwitchTrackOnBackground" Value="{DynamicResource PrimaryHueMidBrush}" />
+    <Setter Property="wpf:ToggleButtonAssist.SwitchTrackOffBackground" Value="{DynamicResource MaterialDesign.Brush.Card.Background}" />
```

Comment:
It seems odd that a `Card` specific background is used here? Should it be `ToggleButton` or a more generic key instead?

```suggestion
    <Setter Property="wpf:ToggleButtonAssist.SwitchTrackOffBackground" Value="{DynamicResource MaterialDesign.Brush.Background}" />
```

Reply:
Fixed